priv/static/mods/pao_rinshan.jq

```diff
@@ -0,0 +1,26 @@
+.after_call.actions += [
+  ["when", 
+    [
+      {"name": "status", "opts": ["rinshan_pao_possible"]},
+      {"name": "match", "opts": [["last_call"], [[[["quad"], 1]]]]}
```

Comment:
rather than set rinshan_pao_possible in daiminkan and check it here, it should be enough to check "kan" status (to verify that we just called kan) and then match last_call with [[[["daiminkan"], 1]]]